src/ripple/consensus/Consensus.h

```diff
@@ -867,15 +867,18 @@ Consensus<Derived, Traits>::timerEntry (NetClock::time_point const& now)
     {
         using namespace std::chrono;
 
+        // checkLCL may change state_, so it needs to happen outside the
+        // cases below.
```

Comment:
Perhaps this comment would be more accurate if it read, "checkLCL may change state_, so it needs to run prior to doing the switch (state_) below."